tests/spiegelman_material.prm

```diff
@@ -0,0 +1,4 @@
+# This test checks wether the derivatives of the simple nonlinear
+# material model are the same as the finite difference derivatives 
+# created by the simple nonlinear material model.
+set Additional shared libraries = tests/libsimple_nonlinear.so
```

Comment:
Please update test description and replace the parameter by:
```
 set Additional shared libraries = libspiegelman_material.so 
```
Technically the test would work without the parameter anyway (because it is automatically added by the test system), but it makes more clear what is happening.

Reply:
sure